libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -202,6 +202,19 @@ void AP_Mount_Backend::set_roi_target(const Location &target_loc)
     }
 }
 
+// set yaw lock - sets the _yaw_lock variable and captures current earth frame heading of mount for targeting in RC Targeting mode
+void AP_Mount_Backend::set_yaw_lock(bool yaw_lock)
+{
+    // if yaw not locked already, capture mount's earth frame heading for later possible use
+    if (!_yaw_lock) {
+        float roll_in, pitch_in, yaw_in;
+        get_rc_input(roll_in, pitch_in, yaw_in);
```

Comment:
Thanks again for this.  I don't think this is quite right because the yaw_in number is going to be a number between -1 and +1 (radians) which may appear to work because that's -57 ~ + 57 degrees but actually I think we need to convert the yaw_in to an angle using this line:

`float yaw_in_rad = radians(((yaw_in + 1.0f) * 0.5f * (_params.yaw_angle_max - _params.yaw_angle_min) + _params.yaw_angle_min));`

Reply:
fixed and tested...rebased on top of Peter's